ENIGMAsystem/SHELL/Universal_System/Extensions/ExternalFuncs/About.ey

```diff
@@ -0,0 +1,13 @@
+%e-yaml
+---
+
+Name: External Functions
+Identifier: ExternalFuncs
+Author: Josh Ventura
+Description: Enables the use of GameMaker's legacy external functions for calling stuff from various DLL's. Requires the libffi dependency.
```

Comment:
I wouldn't call these legacy functions, which has a negative connotation. I usually use the word classic, but in this case I wouldn't even say that, they are still in GMS and they very clearly have a purpose and a use case.